groups.go

```diff
@@ -48,6 +48,15 @@ type Group struct {
 	Projects             []*Project         `json:"projects"`
 	Statistics           *StorageStatistics `json:"statistics"`
 	CustomAttributes     []*CustomAttribute `json:"custom_attributes"`
+	LDAPCN               string             `json:"ldap_cn"`
+	LDAPAccess           int64              `json:"ldap_access"`
+	LDAPGroupLinks       []LDAPGroupLink    `json:"ldap_group_links"`
+}
```

Comment:
Can you point me to something that shows there fields are part of the response when a LDAP group is linked? If there isn't something about this in the docs, maybe the raw JSON API response? Thanks!

Reply:
Here's an example for an example listing of the links

```
[
  {
    "cn": "gitlab_group_example_30",
    "group_access": 30,
    "provider": "example_ldap_provider"
  },
  {
    "cn": "gitlab_group_example_40",
    "group_access": 40,
    "provider": "example_ldap_provider"
  }
]
```